devicetypes/smartthings/hue-bridge.src/hue-bridge.groovy

```diff
@@ -15,19 +15,27 @@ metadata {
 		// TODO: define status and reply messages here
 	}
 
-	tiles {
+	tiles(scale: 2) {
+     	multiAttributeTile(name:"rich-control"){
+			tileAttribute ("", key: "PRIMARY_CONTROL") {
```

Comment:
OK, I'm not sure how concerned we are with older clients, but that is used to display tiles on older clients.